### PR TITLE
I don't even know

### PR DESCRIPTION
**Pull request by elijahkim**
* Changed styling
* User destroy added

### CODE REVIEW

**Comment by HayleyCAnderson** [comment]
nice title

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -18,7 +18,10 @@
 //= require owl.carousel
 //= require mapbox.js
 
-$('.datetimepicker').datetimepicker();
+$('.datetimepicker').datetimepicker({
+  inline: true,
```
Comment (HayleyCAnderson) [inline app/assets/javascripts/application.js]: is this CSS?

File: app/assets/stylesheets/pages/_index.scss [hunk 2]
```diff
@@ -23,11 +23,13 @@
   h1 {
     color: $orange;
     font-size: 5em;
+    -webkit-text-stroke: 1px black;
```
Comment (HayleyCAnderson) [inline app/assets/stylesheets/pages/_index.scss]: if you need the webkit prefix, you probably need the other prefixes too

Comment (HayleyCAnderson) [inline app/assets/stylesheets/pages/_index.scss]: unless bourbon is doing that for you

File: app/models/user.rb [hunk 3]
```diff
@@ -52,6 +53,6 @@ def requested_match(user)
   end
 
   def matches
-    outgoing_matches.append(incoming_matches).order(match_at: :asc)
+    Match.where("challenger_id = :user_id OR defender_id = :user_id", user_id: id).order(match_at: :desc)
```
Comment (HayleyCAnderson) [inline app/models/user.rb]: maybe make the string into a private method

File: app/views/match_histories/show.html.erb [hunk 4]
```diff
@@ -1,12 +1,15 @@
 <div class="base-container">
 <h1 class="h1-banner">Match History</h1>
 
+<% if @matches.empty? %>
```
Comment (HayleyCAnderson) [inline app/views/match_histories/show.html.erb]: why aren't you using a partial or something?

File: app/views/pages/index.html.erb [hunk 5]
```diff
@@ -18,7 +18,7 @@
     </div>
     <div>
       <div class="banner-text">
-        <h1>See Tangable Growth</h1>
+        <h1>See Tangible Growth</h1>
```
Comment (HayleyCAnderson) [inline app/views/pages/index.html.erb]: :+1:

File: app/models/user.rb (hunk 3 above)
Comment (elijahkim) [inline app/models/user.rb]: I end up calling it in a bunch of controllers. I'm not sure if they could access the method if its private

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (elijahkim) [inline app/assets/javascripts/application.js]: Its from the datetimepicker API. I think its some javascript. I have no idea

http://xdsoft.net/jqplugins/datetimepicker/

File: app/models/user.rb (hunk 3 above)
Comment (HayleyCAnderson) [inline app/models/user.rb]: I'm just talking about the string

File: app/assets/stylesheets/pages/_index.scss (hunk 2 above)
Comment (elijahkim) [inline app/assets/stylesheets/pages/_index.scss]: I'm not sure what you mean here.

File: app/views/match_histories/show.html.erb (hunk 4 above)
Comment (elijahkim) [inline app/views/match_histories/show.html.erb]: Because I'm lazy. I'll do it :(

File: app/assets/stylesheets/pages/_index.scss (hunk 2 above)
Comment (HayleyCAnderson) [inline app/assets/stylesheets/pages/_index.scss]: -webkit- is a vendor prefix. It makes it work on chrome and safari, but it won't work on firefox and opera unless you add -moz- and -o-. Unless bourbon does all the vendor prefixes for you, in which case you might not even need the -webkit-

File: app/models/user.rb (hunk 3 above)
Comment (elijahkim) [inline app/models/user.rb]: Oh cool yeah. Thanks

File: app/assets/stylesheets/pages/_index.scss (hunk 2 above)
Comment (elijahkim) [inline app/assets/stylesheets/pages/_index.scss]: I'll check the bourbon docs. Thanks

**Pull request merged**